src/main/java/com/dao/homework/dao/jdbc/CarDaoJdbcImpl.java

```diff
@@ -177,9 +170,11 @@ public Manufacturer getManufacturerInstance(ResultSet resultSet) throws SQLExcep
     }
 
     public Driver getDriverInstance(ResultSet resultSet) throws SQLException {
-        Driver driver = new Driver(resultSet
-                .getObject("drivers_name", String.class),
-                resultSet.getObject("license_number", String.class));
+        String name = resultSet.getObject("drivers_name", String.class);
+        String licenseNumber = resultSet.getObject("license_number", String.class);
+        String login = resultSet.getObject("login", String.class);
+        String password = resultSet.getObject("password", String.class);
```

Comment:
ideally, we don't want to expose users' password when passing an object to a front end